### ISSUE TITLE
How to do_predict on run_glue?

### ISSUE THREAD

**Issue by azamatolegen** [huggingface/transformers#2262]
## ❓ Questions & Help
I have fine-tuned BERT for sequence classification task by running run_glue script. Now I have trained and evaluated model. My question is how do I make prediction with it on new instances (test set)? Thanks!

**Comment by JetRunner** [comment]
#2198 Same here. A predict script can be really helpful to researchers.

**Comment by yuchenlin** [comment]
same here
